data/json/items/toy.json

```
@@ -31,6 +67,192 @@
         "description": "A large plush ferret with cute beady eyes."
       },
       { "id": "toy_plush_trex", "name": { "str": "T-rex plushie" }, "description": "A large plush t-rex." },
+      {
+        "id": "toy_plush_trice",
+        "name": { "str": "triceratops plushie" },
+        "description": "A large, round, cuddly dinosaur plushie with soft horns and a sturdy build.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_stego",
+        "name": { "str": "stegosaurus plushie" },
+        "description": "A large stegosaurus plushie with a spiky back and soft, huggable body.  Don't worry, it is completely docile!",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_ankylo",
+        "name": { "str": "ankylosaurus plushie" },
+        "description": "A cute, squat ankylosaurus plushie with a spiky tail and soft plating.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_vel",
+        "name": { "str": "velociraptor plushie" },
+        "description": "A furry velociraptor plushie with oversized eyes and little claws.  It has a very friendly demeanor.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_diplo",
+        "name": { "str": "diplodocus plushie" },
+        "description": "A long and floppy diplodocus plushie with a gentle expression and wrinkled skin.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_brachio",
+        "name": { "str": "brachiosaurus plushie" },
+        "description": "A sweet-faced brachiosaurus plushie with a long neck and oversized head.  It looks very wise!",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_spino",
+        "name": { "str": "spinosaurus plushie" },
+        "description": "A spiky spinosaurus plushie with big claws and a friendly demeanor, he is holding a fish, probably saving it for later.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_pteranodon",
+        "name": { "str": "pteranodon plushie" },
+        "description": "A large pteranodon plushie with big wings and adorable eyes.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_allo",
+        "name": { "str": "allosaurus plushie" },
+        "description": "A fierce-looking allosaurus plushie with a soft, huggable body and clean white teeth.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_compo",
+        "name": { "str": "compsognathus plushie" },
+        "description": "A soft compsognathus plushie, with a cute and bright appearance like that of a baby dinosaur (aren't compsognathus literal baby dinos?).",
```

```suggestion
        "description": "A soft compsognathus plushie, with a cute and bright appearance like that of a baby dinosaur.",
```
They are a distinct species 😛